SEED-platform/seed
# 1716 hand edit notes

#### Any background context you want to provide?
See attached issue.

#### What's this PR do?
Creates a new log note every time a record is edited. Also, log notes are rendered on the inventory detail note page.

#### How should this be manually tested?
Edit records and see that notes are created for each edit.

For code review, you can walk through each commit as the changes within each are mutually exclusive.

**Note:** Log notes can't be edited or deleted via the UI. Technically, they can still be deleted via the API. I could see that the editing or deletion of these notes are strongly discouraged. But, it seemed too rigid to entirely prevent the editing or deletion of these notes, so this is the solution I came up with. Thoughts?

#### What are the relevant tickets?
#1716 

#### Screenshots (if appropriate)
![Screen Shot 2020-04-27 at 5 36 57 PM](https://user-images.githubusercontent.com/30608004/80432405-6139fa80-88b1-11ea-8755-801415c15ae2.png)
